src/iuse.cpp

```diff
@@ -5556,7 +5562,7 @@ int iuse::portal(player *p, item *it, bool, const tripoint& )
     if (it->charges < it->type->charges_to_use()) {
         return 0;
     }
-    g->m.add_trap(p->posx() + rng(-2, 2), p->posy() + rng(-2, 2), tr_portal);
+    g->m.add_trap({p->posx() + rng(-2, 2), p->posy() + rng(-2, 2), p->posz()}, tr_portal);
```

Comment:
> Entertainingly, none of the other Jenkins builds complained. I still find that odd.

`long int` is probably the same as `int` on 32bit systems, so there is no narrowing. Maybe the `point`/`tripoint` class should use `long`s instead?